script/build

```diff
@@ -41,6 +41,21 @@ def check_cxx_version() -> None:
     """
     Check that the user's compiler is compatible with ERT.
     """
+    # we treat silly debian first
```

Comment:
Rewrite to fit the language style of the rest of the codebase.

Reply:
it is done.